tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -6,6 +6,22 @@ ARG IMAGE_REGISTRY=docker.io
 FROM ${IMAGE_REGISTRY}/ubuntu:@OS_VERSION@
 @SET_PROXY@
 
+ENV GO_HOME="/opt"
+ENV GOCACHE="${GO_HOME}/.cache"
+ENV RUSTUP_HOME="/opt/rustup"
+ENV CARGO_HOME="/opt/cargo"
+ENV PATH="/opt/cargo/bin/:/opt/go/bin:${PATH}"
+
+ARG GO_VERSION=1.22.2
+ARG RUST_TOOLCHAIN
```

Comment:
Where is the `RUST_TOOLCHAIN` passed in?

Reply:
rootfs.sh:build_rootfs_distro
```
	"$container_engine" build  \
			${engine_build_args} \
			--build-arg http_proxy="${http_proxy}" \
			--build-arg https_proxy="${https_proxy}" \
			--build-arg RUST_TOOLCHAIN="$(get_package_version_from_kata_yaml  "languages.rust.meta.newest-version")" \
			-t "${image_name}" "${distro_config_dir}"
```

Reply:
Looks like I forgot to update the `roofs.sh` :) I will also get the GO version from versions.yaml and update this PR, good catch!